src/components/ContactItem/ContactItem.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import styles from './ContactItem.module.scss';
+import Modal from "../Modal/Modal";
+import ContactModalBody from "../ContactModal/Body";
+
+interface Props {
+    id: number,
+    name: string,
+    phone: string,
+}
+
+const ContactItem = (props: Props) => {
+    const { id, name, phone } = props;
+    const [showItemModal, setShowItemModal] = useState(false);
```

Comment:
Ar fi mai ok sa muti state-ul modalului si locatia componentei de modal pentru "Contact info" in ContactList. Pentru ca nu vrei sa ai mai multe modale de contact details in acelasi timp. In felul asta ai lasa si ContactItem ca o componenta pura.